### PR TITLE
ldconfig: add page

### PR DESCRIPTION
**Pull request by agnivade**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

File: pages/linux/ldconfig.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# ldconfig
+
+> Configure symlinks and cache for shared library dependencies.
+
+- Update symlinks and rebuild the cache. (Usually run when a new library is installed):
```
Comment (sbrl) [inline pages/linux/ldconfig.md]: Don't think the full stop is needed here. Also, I'm not sure that `Usually` needs a capital letter at the beginning.

File: pages/linux/ldconfig.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+`sudo ldconfig`
+
+- Update symlinks for a given directory:
```
Comment (sbrl) [inline pages/linux/ldconfig.md]: Perhaps `Update symlinks` -> `Update the symlinks` reads better?

File: pages/linux/ldconfig.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+`sudo ldconfig -n {{path/to/directory}}`
+
+- Print the libraries in the cache and check whether a given library is present:
```
Comment (sbrl) [inline pages/linux/ldconfig.md]: Perhaps just including the `ldconfig -p` bit would be ok? Not sure.

Comment (agnivade) [reply pages/linux/ldconfig.md]: You would almost always never use `ldconfig -p` without wanting to check for a particular library. This is how the -p flag is commonly used and in the interest of putting useful and practical examples, I have added the grep line.

File: pages/linux/ldconfig.md (hunk 1 above)
Comment (agnivade) [reply pages/linux/ldconfig.md]: done

File: pages/linux/ldconfig.md (hunk 3 above)
Comment (sbrl) [reply pages/linux/ldconfig.md]: Ahh ok. Looks good to me then!

**Pull request merged**